.github/workflows/dco-check.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         if: always()
         run: |
           mkdir -p ./pr
-          echo -n ${{ steps.reverted_pr_check.outputs.is_reverted_pr }} > ./pr/RPR
+          echo -n ${{ steps.reverted_pr_check.outputs.is_reverted_pr }} > ./pr/s_reverted_pr
```

Comment:
typo here

Reply:
Sorry, overlooked, Fixed it now @quentinDERORY